---
title: "About me"
---

![](images/clipboard-496297121.png){width="400"}

Hello! My name is Nalany Richardson and I am a 2nd year Masters student in the University of Georgia's Comparative Biomedical Sciences department, in the College of Veterinary Medicine, in the Dan Peach Lab. I am currently based at the <b>Savannah River Ecology Lab</b> in Jackson, South Carolina.

I am a 2nd year Masters student in the University of Georgia's Comparative Biomedical Sciences department, in the College of Veterinary Medicine, in the Dan Peach Lab. I am currently based at the <b>Savannah River Ecology Lab</b> in Jackson, South Carolina.

## Current Research and Interests ðŸ¦Ÿ ðŸ›‘

My current research looks to determine how ornithophilic ('bird-loving') mosquitoes such as <i>Culiseta melanura</i> and <i>Culex spp.</i> specifically seek out avian hosts as these interactions can lead to transmission of highly lethal viruses such as Eastern Equine Encephalitis Virus (EEEV). By utilizing volatile organic compound (VOC) blends from dynamic headspace aeration, my research group is looking to <b>develop cost-effective, targeted, and field-applicable approaches for mosquito control and surveillance. </b>

## Statistics and Analysis Background

I developed an interest in bioinformatics in my undergraduate lab, where I used bash scripts with my data from <b>Geneious, Clustal Omega and Ensembl</b> to develop primers and target amplification of single nucleotide polymorphisms (SNPs) on significant genes of sockeye salmon (<i>Oncorhynchus nerka</i> associated with changes in egg and body color across different regions of Alaskan lakes and beaches.

At the same time, I enrolled in an introductory bioinformatics course at the University of South Carolina-Aiken that was partnered with the <b>American Campus Tree Genomes project</b> led by Dr. Alex Harkess at HudsonAlpha â€” where we focused on the Sabal palmetto. I learned how to write scripts and <b>analyze genome sequencing, assembly, and annotation data using basic linux and shell scripting, as well as create a Circos diagram for our manuscript.</b>

I enrolled in Bioinformatics and Omics in the fall of 2025, and learned how to utilize R and MATLAB to do a great deal of high dimensional data analyses, such as **tree-building, bootstrap analysis, power analyses, BATMAN fitting**, etc... and ran **BUSCO analyses** for my term project to uncover genome completeness assembly robustness across different genera of mosquitoes.

TL;DR: I consider myself a novice, though I can find my way around a terminal and execute scripts and code so long as the documentation is clean (*though... when is it ever??*). Below is a list of my most commonly used softwares and tools for data analysis.

-   **R/R Studio (for all my basic bar graphs)**

-   **MATLAB (my favorite for PCA and anything involving highly dimensional data)**

-   **Bash (for organizing and executing scripts on my data or other big public data sets)**

## Goal for MADA

I want to become more confident in my decisions for which softwares/tools and which analyses to run for data sets that are multivariable and quite franklyâ€” a pain to tidy up and analyze. I hope that through the projects completed in this class, I can become a better resource to not only myself but my lab, and rely less on others to understand my own data.

## Fun facts about me!

-   Crochet is a passion hobby of mine, and love to make headphone accessories and beanies for friends and family.

-   I have 2 shih tzus, Toast (6yo) and Bandit (6mo), who act as my organic alarm clocks.

    ![](images/clipboard-4147792134.jpeg){width="200"}![](images/clipboard-1382437085.jpeg){width="200"}

-   I'm getting married this upcoming Februrary 28th! ðŸ’’

## Link

I would like to share instructions given by the American Campus Tree Genomes people for running [a haplotype resolved chromosome scale assembly](https://actg-wgaa.readthedocs.io/en/latest/Module_03/Lesson_02/lab.html). Forewarning, this is also a quarto blog and YES there are missing pieces. However, the pieces that are there are good. Cheers!
